pkg/users/account-create-dialog.js

```diff
@@ -306,7 +397,7 @@
                     caption: _("Create account with weak password"),
                     style: "warning",
                     clicked: () => {
-                        return passwd_check(true, state.real_name, state.user_name, state.password, state.password_confirm, state.locked);
+                        return passwd_check(true, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.home_dir, state.base_home_dir);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230131-135835-5b43f8e5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)